Automattic/wp-calypso
# Purchase Management: Add a disconnected state warning to purchase page

#### Proposed Changes

These changes pertain to 'part 2' of this issue https://github.com/Automattic/payments-shilling/issues/1321

This PR will add a `renderDisconnectedStateWarning()` function that will render a card in place of the Cancel/Remove button when a purchase is no longer connected to a site.

#### Testing Instructions

- This will require two wpcom accounts and a self-hosted site, I recommend using https://jurassic.ninja
- Have account A create a site and connect Jetpack
- Have account A invite account B as an admin from the Calypso dashboard
- Have account B purchase a Jetpack Complete plan
- Have account A remove account B
- While logged into account B, go to your purchases at /me/purchases and select the Jetpack Complete plan you bought, you should no longer be an admin on this site anymore but you **should** still own the plan
- When you click on the Jetpack Complete plan to open the individual plan management page, you should no longer see a cancel or remove button at the bottom, but instead a message like:

![image](https://user-images.githubusercontent.com/16580129/215224716-367c7f17-7ab9-43ed-9651-dc6885520f6b.png)


#### Pre-merge Checklist





Related to https://github.com/Automattic/payments-shilling/issues/1321
